src/opnsense/mvc/app/controllers/OPNsense/TrafficShaper/forms/dialogPipe.xml

```diff
@@ -47,14 +47,14 @@
     </field>
     <field>
         <id>pipe.codel_target</id>
-        <label>(FQ-)CoDel target</label>
+        <label>(FQ-)CoDel Target</label>
         <type>text</type>
         <advanced>true</advanced>
         <help>Minimum acceptable persistent queue delay (in ms), leave empty for default</help>
     </field>
     <field>
         <id>pipe.codel_interval</id>
-        <label>(FQ-)CoDel interval</label>
+        <label>(FQ-)CoDel Interval</label>
```

Comment:
this too

Reply:
System Information widget:
CPU Type
MBUF Usage

Services widget:
Gateway Monitoring Daemon
System Configuration Daemon
Users and Groups
Network Time Daemon
Secure Shell Daemon
Packet Filter

Would be nice if you guys would make up your mind.  ;)